packages/veto-backend/src/api/cyphernode/bumpFee.ts

```diff
@@ -0,0 +1,16 @@
+import { AxiosInstance, AxiosResponse } from 'axios'
+import cyphernodeClient from './client'
+
+const cyphernode: AxiosInstance = cyphernodeClient()
+
+type Args = {
+  txid: string
```

Comment:
:thinking: wondering if its a `tx id`, if so I think it should be in camel case for consistency even if cyphernode define it as `txid`